src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.cs

```diff
@@ -645,6 +647,30 @@ public bool WriteIndented
             }
         }
 
+        /// <summary>
+        /// Defines the text used as indent when <see cref="WriteIndented" /> is enabled
+        /// By default, the JSON is indented with 2 white spaces.
+        /// </summary>
+        /// <exception cref="InvalidOperationException">
+        /// Thrown if this property is set after serialization or deserialization has occurred.
+        /// </exception>
+        public string IndentText
+        {
+            get
+            {
+                return _indentText;
+            }
+            set
+            {
+                if (value is null)
+                {
+                    ThrowHelper.ThrowArgumentNullException(nameof(value));
+                }
```

Comment:
Same here, shouldn't we validate that the string is all-whitespace?

Reply:
I would recommend we stick with what the JSON specification allows, so any combination of space and horizontal tab should be allowed. I would probably leave out CR and LF though since it negates the purpose of indentation.

cc @bartonjs for transparency.